File: mcradar/db/player.go
package db

import (
	"time"

	"gorm.io/gorm"
)

type Player struct {
	gorm.Model
	Id        string `gorm:"primaryKey"`
	CreatedAt time.Time
	UpdatedAt time.Time
	DeletedAt gorm.DeletedAt `gorm:"index"`
	Name      string
}

File: mcradar-api/app/models/player.py
from pydantic import BaseModel, ConfigDict
from pydantic.alias_generators import to_camel
from datetime import datetime

class Player(BaseModel):
    model_config = ConfigDict(alias_generator=to_camel, populate_by_name=True)

    id: str
    created_at: datetime
    updated_at: datetime
    name: str